frontend/packages/tools/workflow-registrar.js

```diff
@@ -39,7 +39,7 @@ const discoverWorkflows = () => {
     return childProcess.exec(
       `yarn list --json --depth=0 --pattern '${packagePattern}'`,
       {
-        cwd: ".",
+        cwd: `${process.argv[2]}/..`,
```

Comment:
can you pull it into a named var so it's clear what it is? a little terse otherwise.

Reply:
comment is okay too